basket/settings.py

```diff
@@ -29,6 +27,7 @@ def path(*args):
 
 
 DEBUG = config("DEBUG", default=False, cast=bool)
+TESTING = config("TESTING", default=False, cast=bool)
```

Comment:
What is this setting for? Only when unittests are being run, or something else?

Reply:
Yes just for unit tests

Reply:
Any reason to have it be a setting like this rather than be detected like we did at the bottom of the file? Or is there some mechanism that will set a `TESTING` env var when we're running tests?

Reply:
I added it to `docker/envfiles/test.env` and at one point I thought I added it to the pytest env settings for running tests outside of Docker, but I don't see that at the moment. I found the settings changes at the bottom of the file when run with certain commands to be less explicit and "hidden" -- I hadn't actually ever noticed that before, as compared to keeping these settings near each other with the `TESTING` conditional. If you prefer it like before I can change it back.

Reply:
I remember adding `pytest-env` and getting that to work, but then I removed it. The reason why is I have never actually run basket outside of Docker due to the `mysql-client` requirement and not wanting to install mysql on my host machine when we have a fully capable docker environment. And since this PR is adding a Redis requirement also, I don't see any reason to support running outside of a Docker environment.

Reply:
True. If it works it's okay with me. I was just thinking something more like:

```python
TESTING = False
if sys.argv[0].endswith("py.test") or sys.argv[0].endswith("pytest") or (len(sys.argv) > 1 and sys.argv[1] == "test"):
     TESTING = True
```

Just to avoid having tests fail if we forget to set an env var during tests. While running the system does require everything, just running unit tests shouldn't. I tend to run them locally.

Reply:
I added this in and changed the setting name to `UNITTEST`.